### ISSUE TITLE
Not really a issue.

### ISSUE THREAD

**Issue by FlingeR** [jasperla/openbsd-wip#103]
Sorry, 
I think that the change in the clocks between systems affects heavily on github.
I say this because I checked the repository and it said that I was on sync, then when pushing give me a warning of conflict, when pulling didn't said anything, and it ended up in a "merged master" thingy.

It seems that @fcambus pushed changes (according to this site "2 hours ago") and my machine is 3 hours behind (GMT-3), so according to MY GIT there where no changes (because changes where an hour ahead????) and now appears my push, the push of fcambus and a merging branch, because according to MY GIT, that is how things happened. xD

Just to explain if this affected something else.
Not a big git user myself...

What a horrible thing this uncentralized system...

**Comment by mazocomp** [comment]
@FlingeR hm, shouldn't this also be sent to both Github and Linus Torvalds/Junio Hamano/whoever?

BTW, I wish this repo was CVS too, I can't stand Git and its useability.

**Comment by FlingeR** [comment]
@mazocomp 
You forgot about Microsoft, the new owner of Github...
But is not all bad or all good... I think that the problem with all the CVS systems that I used is that they are not very clear in what is going to happen,
I mean, take a look at the [commits list](https://github.com/jasperla/openbsd-wip/commits/master).
I pushed my changes after fcambus and what happend is that I leterally swap the order, even my merge is "before" his push.
That's freaky... And all I did was:
git checkout
(nothing...)
git push origin master
(conflict found)
git checkout
(nothing...)
git pull
(retrived the changes from github)
git push origin master
(you have just merged your git repository with the github one)
WHAT? I did not ask for that!

Just changed things to use UTC to avoid further problems.
